multimodal/src/autogluon/multimodal/utils/__init__.py

```diff
@@ -49,8 +49,9 @@
     from_voc,
     get_detection_classes,
     get_image_name_num,
-    get_voc_classes,
+    get_voc_format_classes,
```

Comment:
we shall use get_detection_classes which calls get_voc_format_classes or get_coco_format_classes depending on if input path is a dir or file.